lib/telemetry_metrics_statsd.ex

```diff
@@ -402,6 +408,7 @@ defmodule TelemetryMetricsStatsd do
 
   @impl true
   def init(config) do
+    Process.flag(:trap_exit, true)
```

Comment:
Why do we need to trap exits now?

Reply:
Sorry, I didn't notice that it's been moved from down below! Of course we need it, thanks!